src/main/webapp/sell.html

```diff
@@ -41,9 +41,31 @@
     <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.min.js"
         integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
         crossorigin="anonymous"></script>
+    <!--Google Map API-->
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAuKSxxOHI1o-tBCm-wS1sqDyneVxbKAlM&libraries=places&callback=createMap"></script>
+    <script src="/js/sellMap-loader.js"> </script>
+    <style>
```

Comment:
Maybe you can move this to a separate css page to make the code cleaner

Reply:
made a new commit to add  a css file and remove those code